guides/v2.1/contributor-guide/contributing.md

```diff
@@ -38,6 +38,14 @@ Often when the Community Engineering Team works on reviewing the suggested chang
 <p>Please refer to <a href="http://www.magento.com/legaldocuments/mca">Magento Contributor Agreement</a> for detailed information about the License Agreement. All contributors are required to submit a click-through form to agree to the terms. </p>
 </div>
 
+## GitHub and Two-Factor Authentication {#two-factor}
+
+When setting up access and tokens for Magento GitHub repositories, we recommend adding Two-Factor Authentication (2FA) to enhance security. Magento **requires all Partners** who contribute code to enable Two-Factor Authentication (2FA) on their accounts. You can use a mobile device or 2FA application for added protection.
+
+Two-factor authentication adds an additional layer of security beyond just a username/password when you access GitHub. With 2FA enabled, you log into a service with your credentials then complete an additional step providing a 2FA code. This second form of authentication helps us ensure that a malicious user will not be able to gain access to your GitHub account or any private Magento GitHub repositories.
```

Comment:
Everywhere else you call it "Two-Factor Authentication" but here it's "Two-factor authentication" (I prefer this one). Make consistent?